### PR TITLE
fix(bazel): specify the package and lock files using the workspace 

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
When used downstream (CLI), The following error is being thrown
```bash
no such package '@ts-api-guardian_runtime_deps//': Unable to load package for //tools/ts-api-guardian:package.json: not found. and referenced by '//etc/api:angular_devkit_schematics_tasks_api'

no such package 'tools/ts-api-guardian': BUILD file not found on package path and referenced by '//etc/api:angular_devkit_schematics_testing_api.accept'
```

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

//cc @alexeagle

### CODE REVIEW

File: tools/ts-api-guardian/index.bzl [hunk 1]
```diff
@@ -17,14 +17,12 @@
 
 load("@build_bazel_rules_nodejs//internal/node:node.bzl", "nodejs_binary", "nodejs_test")
 
-COMMON_MODULE_IDENTIFIERS = ["angular", "jasmine", "protractor"]
```
Comment (alexeagle) [inline tools/ts-api-guardian/index.bzl]: I'd rather keep this constant (it's technically part of the public API for this rule)
and use it below as the default for the named param

**Review by alexeagle — APPROVED** [review]

File: tools/ts-api-guardian/index.bzl (hunk 1 above)
Comment (alexeagle) [reply tools/ts-api-guardian/index.bzl]: I went ahead and made the change so we can get this greened up

**Comment by alan-agius4** [comment]
Thanks @alexeagle!

**Comment by IgorMinar** [comment]
This PR doesn't apply cleanly to the patch branch. @alan-agius4 can you send a separate PR for that branch please?

**Comment by alan-agius4** [comment]
Yeah I can do that tomorrow. Most likely it will have to contain the
changes of the following PR as well
https://github.com/angular/angular/pull/25690 as it was merged only to
master.

**Pull request closed without merging**